eliteSchwein/mooncord
# docker

- update: move image path
- update: prepare webpack
- update: add dev build and include sharp dependency
- update: add webpack test
- update: replace webpack test
- debug: config
- update: exclude scripts directory
- test: config path
- test: replace methode
- test: only load config
- test: debug path
- test: ignore require
- test: replace require with import
- test: replace require with import
- update: add watch
- test: read file and parse as json
- test: debug json string
- test: disable exit
- update: add progress
- test: debug if fs read file is even running
- test: debug fs
- test: debug read methode
- test: debug read methode
- test: debug read config
- test: debug read config
- test: debug read config
- test: debug read config
- test: debug read config
- test: replace read function
- test: parse config data
- update: replace config reader
- update: fix imports
- update: fix misc util
- update: debug config
- update: rebuild app
- update: fix thumbnail util
- update: fix thumbnail util
- update: fix webcam util
- update: fix webcam util
- update: move import
- update: move import
- update: delete all Javascript coded stuff, prepare for typescript
- test: debug to console
- test: update paths
- update: replace console colors library
- update: remove old packages, add console class
- update: update default color and add empty post
- add: confighelper
- add: prepare new moonraker client
- remove systeminformations
- remove fluent ffmpeg
- add apikey helper
- update: add websocket connection
- update: add initial websocket commands
- update console Log
- update: make oneshottoken getter sync
- update: change websocket dependency
- update: fix APIKeyHelper, fix initial events, fix command executer
- update: merge functions
- update: merge functions
- update: fix subscription command
- update: remove console logger class and just export the functions
- update command handler
- update: rework command handler, add custom timeout
- update: add config section for moonraker database namespace
- update: add database util
- update: add more Logs, prepare CacheUtil
- update: add Dump Methode
- update methodes
- update: replace webpack with ncc, cleanup compiler configs and packages
- update: add functions to the discord Client
- prepare discord command Generator
- prepare discord command Generator
- update: add multi language package
- update: add multi language support, remove i18n
- update: add dump command template, register generated commands
- update log code
- minify code
- update console log
- update dependencies
- update: add 2 more startup commands
- update: remove proc stats initial command
- update: remove loadinfo structure, add systeminfo structure, add dump file, autodump for now, add mindmap word document
- update mindmap
- update: add systemcomponents getter methode
- update: expand permission section, remove deprecated sections
- update: add Proc Stats and SubscriptionUpdate Events for Moonraker Connection
- update: rename notification handler, add update notification handler
- update: cache generated commands
- update: add button generator and metadata
- bugfix: add default value per button
- update: add permissions and permission mapping for buttons
- update: add single button generator
- update: prepare selection mapping
- update: rename button generator, implement Selection Generator
- update: rename variables and methodes
- update: add default config merge
- update: use bare minimum config for install script
- update: remove unused import
- update: prepare InteractionHandler
- update: add debug event handler, cache basic clients informations
- update: add fallback locale loader
- update: move locales to Cache
- update: add Interaction Handlers
- update: testing with Presence and Game Activity
- update: add Status Message Generator
- update: add fields title to status Generator
- update locale structure, WARNING its not implement into the Code yet
- update: implement new locale structure
- update: add command ID getter, expand full config with custom info section, fix status mapping embed ids
- update: add value find methode, disable update refresh, add dump Command, fix class Argumens for Command Interactions, add package Config to Cache
- add Log for Command executions
- update packages
- update packages
- update: add permission Help, add visible config for no permission message, add loadCache for config and locale helpers, remove useless methodes, remove duplicated no permission messages
- update: add embed helper with basic placeholder parser
- update: add embed thumbnail and button parser, remove useless button assign meta
- update: slim down Cache Generator, update button locale structure
- update: add Temp Helper and Formatting Helper
- update: add temp mapping, add temp command
- update: add check if buttons row is empty, remove faulty functions
- update: change fan symbol, add min and max temp
- update: add cold state
- update: move answer
- update: add Restart Command, add Fallback Answer on Command Handler
- update: log complete command
- update: prepare get Log Command
- update: add log file Handler, rename Logger Helper
- update: add log level, add ascii strip methode
- update packages
- update: prepare slow fan icon
- update: add get user id command
- update: add Database Reset Command and Methode
- update: rename ObjectMerge Helper, add Database Dump, rename Cache Dump File, optimize Command Locale Structure
- bugfix: Database didn`t update
- update: remove default value for notify list
- update: write invite url to database, remove namespace config
- update: add emergency stop command
- Add a Dockerfile to support building containers
